CHANGELOG.next.asciidoc

```diff
@@ -55,6 +55,7 @@ https://github.com/elastic/beats/compare/v8.2.0\...main[Check the HEAD diff]
 
 - Fix counter for number of events published in `httpjson` input. {pull}31993[31993]
 - Fix handling of Checkpoint event for R81. {issue}32380[32380] {pull}32458[32458]
+- Fix a hang on `apt-get update` stage in packaging. {pull}32580[32580]
```

Comment:
```suggestion
```

I don't think we need a changelog entry for this because it isn't user facing. This could go in the developer changelog though.